source/Mlos.Proto/OptimizerService.proto

```diff
@@ -18,10 +18,18 @@ service OptimizerService
     //
     rpc GetOptimizerConvergenceState(OptimizerHandle) returns (OptimizerConvergenceState);
 
+    // Returns the computed goodness of fit metrics.
+    //
+    rpc ComputeGoodnessOfFitMetrics(OptimizerHandle) returns (SimpleString);

```

Comment:
Add 2 arguments: observations, and data set type.

Actually - just open a work item for it.